### ISSUE TITLE
How up to date is Bootstrap 4 support?

### ISSUE THREAD

**Issue by tkurki** [vazco/uniforms#419]
I am looking into replacing https://github.com/mozilla-services/react-jsonschema-form in https://github.com/SignalK/signalk-server-node/ with Uniforms. SK server supports plugins, that provide the structure of their configuration data in JSON Schema and the server generates the configuration UI from that.

Initial impression is that this might work and Uniforms appears to have solved the problem that plagues RJSF, mainly [being stuck with Bootstrap 3](https://github.com/mozilla-services/react-jsonschema-form/issues/899).

However I bumped against https://getbootstrap.com/docs/4.0/migration/#labels-and-badges in [ListAddField](https://github.com/vazco/uniforms/blob/master/packages/uniforms-bootstrap4/src/ListAddField.js#L18) and [ListDelField](https://github.com/vazco/uniforms/blob/master/packages/uniforms-bootstrap4/src/ListDelField.js#L19).

That would be easy enough to fix and I can PR that, but this raises the question about how up to date the Bootstrap 4 version is and how dedicated Uniforms is to support Boostrap 4 and the various frameworks. It appears the label-badge renaming took place over [2 years ago](https://github.com/twbs/bootstrap/pull/21565/files).

**Comment by radekmie** [comment]
Well, it's a hard question.

At first, I'd love to keep all core packages (this repo) up-to-date. Personally, I'm using mainly Material and Semantic UI themes (last one especially). Most projects I've worked in/seen/heard of are using a custom theme (or at least few custom replacements of provided components). I know about at least two production projects using Bootstrap 4 theme - I guess they have own `ListField` replacement (or there are no lists at all).

So, let's answer your questions (OK, more general ones). How up-to-date are uniforms? As much as people using it report issues (yes, I'm one of these people too). How is dedicated support planned? I'm trying to do my best here: either help with patching any bugs and incompatibilities or fix it by myself. Bootstrap 4 is somehow special here, as each non-final version had some problems. @janslowik made a bigger update in #386, but this seems to be not included. There's only one thing - if I'm not aware of something, I won't fix it.

TL;DR;
Report issues, submit PRs and we are up-to-date.

**Comment by tkurki** [comment]
Well, answering questions like mine is one measure of the level of support. Thanks!

Only I did not understand what you meant by _this seems to be not included_?

**Comment by radekmie** [comment]
I meant that this PR fixed many things but not the labels.

**Comment by tkurki** [comment]
Gotcha!

**Comment by radekmie** [comment]
Are you willing to send a PR with a fix or should I treat this issue as a bug report?

**Comment by tkurki** [comment]
Whichever, I thought I would dig a little deeper to see if there is anything in addition to that. Easier if you fix it...

**Comment by radekmie** [comment]
@tkurki: OK. I'm a bit busy at the moment, so give it a shot - it looks like an easy one to test. I'll surely have some time this weekend to do it too.
